doc/Command-Reference.md

```diff
@@ -2509,6 +2510,65 @@ This command displays a list of NTP peers known to the server as well as a summa
 
 Go Back To [Beginning of the document](#SONiC-COMMAND-LINE-INTERFACE-GUIDE) or [Beginning of this section](#NTP)
 
+# PFC Watchdog Commands
+
```

Comment:
i would suggest to have a high level description of the feature before the commands description just to be clear how it works and how the configuration affect the feature behaviour. should not be long but clear enough

in addition each parameter has a default configuration, it is highly recommended to provide the default. In case it is platform dependent worth mentioning it.

Reply:
Added wiki reference and default parameters description